x/interchainstaking/keeper/ibc_packet_handlers.go

```diff
@@ -273,7 +274,7 @@ func (k *Keeper) HandleCompleteSend(ctx sdk.Context, msg sdk.Msg, memo string) e
 	// get zone
 	zone, err := k.GetZoneFromContext(ctx)
 	if err != nil {
-		return err
+		return fmt.Errorf("2: %s", err.Error())
```

Comment:
"2" ? should probably use consistent string formatting...
Errorf("#: %s", var) <- better than "#:" + var